src/DynamoDbModel.php

```diff
@@ -68,6 +68,12 @@
      */
     protected $dateFormat = DateTime::ATOM;
 
+    /**
+     * The array of global scopes on the model.
+     *
+     * @var array
+     */
+    protected static $globalScopes = [];
```

Comment:
this is used by `Illuminate\Database\Eloquent\Concerns\HasGlobalScopes` to set/retrieve the global scopes ... not sure why the property isn't just part of the scope itself, but this comes directly from the Eloquent Model and Traits.